build.gradle

```diff
@@ -19,7 +19,7 @@ apply plugin: 'eclipse'
 apply plugin: 'idea'
 apply plugin: 'maven-publish'
 
-version = '0.5.19-dev.7'
+version = '0.5.19-dev.10'
```

Comment:
This change shouldn't have beem commited.
We do these when there's builds, this isn't in a build yet, it'll just mess with our versioning.
If you must absolutely create internal builds for your own testing, please use a different prefix, but don't commit that change.

Reply:
Okay. How would you suggest I go about changing this? Should I undo the commit and then remove that specific section or should I add a new commit where I undo the change?